stories/core/Tabs.stories.tsx

```diff
@@ -6,26 +6,22 @@ import SvgStar from '@itwin/itwinui-icons-react/cjs/icons/Star';
 import { useState } from '@storybook/addons';
 import { Meta, Story } from '@storybook/react/';
 import React from 'react';
-import {
-  HorizontalTabs,
-  HorizontalTabsProps,
-  HorizontalTab,
-} from '../../src/core';
+import { Tabs, TabsProps, Tab } from '../../src/core';
 
 export default {
-  title: 'Core/HorizontalTabs',
-  component: HorizontalTabs,
-  subcomponents: { HorizontalTab },
+  title: 'Core/Tabs',
+  component: Tabs,
+  subcomponents: { Tab },
   args: {
     focusActivationMode: 'auto',
     color: 'blue',
   },
   argTypes: {
     children: { control: { disable: true } },
   },
-} as Meta<HorizontalTabsProps>;
+} as Meta<TabsProps>;
 
-const Template: Story<HorizontalTabsProps> = (args) => {
+const Template: Story<TabsProps> = (args) => {
```

Comment:
Can you make that instead of `Template`, `Default` would be reused? It seems that then it would be enough just to change the type in other stories.

Reply:
Think I need new story using `VerticalTabs`